src/styles/post/UploadImageModal.scss

```diff
@@ -2,6 +2,29 @@
 
 .upload-image-modal {
   @extend %modal;
+
+  .dropzone {
+    text-align: center;
+    padding: 20px;
+    border-width: 2px;
+    border-radius: 5px;
+    border-style: dashed;
+    background-color: #fafafa;
```

Comment:
since the background-color is already in the common stylesheet, can we just use that variable directly?

Reply:
Done!